src/slider/styles.scss

```diff
@@ -173,10 +181,18 @@
     background: awsui.$color-text-status-error;
   }
 
+  &.warning {
+    background: awsui.$color-text-status-warning;
+  }
+
   &.error-active {
     background-color: awsui.$color-background-slider-error-pressed;
   }
 
+  &.warning-active {
```

Comment:
Pressing the slider doesn't seem to be changing the color. It looks like there is supposed to be two different colors for active and non-active warning, is that a bug?